sdk/storage/src/core/clients/storage_account_client.rs

```diff
@@ -31,6 +30,19 @@ pub enum ServiceType {
     Table,
 }
 
+struct AzureEnvironment {
+    blob_endpoint: String,
+}
+
+lazy_static! {
+    static ref AZURE: AzureEnvironment = AzureEnvironment {
```

Comment:
consider naming this AZURE_GLOBAL, https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.management.resourcemanager.fluent.azureenvironment?view=azure-dotnet